AllReadyApp/Web-App/AllReady/Hangfire/Jobs/SendRequestStatusToGetASmokeAlarm.cs

```diff
@@ -15,14 +15,17 @@ public class SendRequestStatusToGetASmokeAlarm : ISendRequestStatusToGetASmokeAl
         public SendRequestStatusToGetASmokeAlarm(IOptions<GetASmokeAlarmApiSettings> getASmokeAlarmApiSettings, IHttpClient httpClient)
         {
             _getASmokeAlarmApiSettings = getASmokeAlarmApiSettings.Value;
+            _httpClient = httpClient;
         }
 
         public void Send(string serial, string status, bool acceptance)
         {
-            var updateRequestMessage = new { acceptance, status };
-
+            var str = JsonConvert.SerializeObject(new { acceptance, status });
             var request = new HttpRequestMessage(HttpMethod.Post, $"{_getASmokeAlarmApiSettings.BaseAddress}admin/requests/status/{serial}");
-            request.Content = new StringContent(JsonConvert.SerializeObject(updateRequestMessage), Encoding.UTF8, "application/json");
+            request.Content = new StringContent(
+                JsonConvert.SerializeObject(new { acceptance, status }), 
+                Encoding.UTF8, 
+                "application/json");
             request.Headers.Authorization = AuthenticationHeaderValue.Parse(_getASmokeAlarmApiSettings.Token);

```

Comment:
I was thinking its more difficult to read when you have parameters with nested functions all in one line.